### ISSUE TITLE
reverese relations section refers to not existing aria-described property

### ISSUE THREAD

**Issue by asurkov** [w3c/core-aam#65]
reverese relations section refers to not existing aria-described property [1], should be aria-describedby

[1] https://w3c.github.io/core-aam/#mapping_additional_relations_reverse_relations

**Comment by joanmarie** [comment]
D'oh! Thanks for catching that. Should be fixed in master and stable branches.
